README.md

```diff
@@ -27,6 +29,67 @@ Scripts to create and load data into the `gxa-*` Solr indexes (for analytics). E
 - jq (1.5)
 - curl
 
+However, the direct usage without `atlas-index-base:1.5` container (or the container tag currently used in the CI tests on [run_tests_in_containers.sh](run_tests_in_containers.sh)) is discouraged.
+
+# Authentication
+
+The setup on the CI is made to use authentication with default user and password. The calls assume these settings (solr:SolrRocks), but the user and password can be modified by doing:
+
+```
+export SOLR_USER=<new-user>
+export SOLR_PASS=<new-pass>
+```
+
+To use default auth in a new solr cloud instance, upload `test/security.json` to ZK as shown in the `Setup auth` part of the `run_tests_in_containers.sh`. To setup users in a production setting the script [create-users.sh](bin/create-users.sh) will receive two set of users:
+
+```
+ADMIN_USER=<admin-username>
+ADMIN_U_PWD=<password>
+
+QUERY_USER=<query-username>
+QUERY_U_PWD=<password>
+```
+
+it will create both users, giving the first admin privileges and the second reading privileges only, delete the default user and set the instance to only work with authenticated users.
+
+## Enable BioSolr
+
+`bulk-analytics-v1` makes use of the [BioSolr plugin](https://github.com/ebi-gene-expression-group/BioSolr) to perform ontology expansion on document indexing based on EFO. In order to enable BioSolr, there are 2 options:
+
+### Option 1: Local `.jar` file
+
+Place BioSolr jar (which can be found in the repository's `lib` directory) under `/server/solr/lib/` in your Solr installation directory. This is the oldest option, and has some security issues, but for testing should be fine.
+
+### Option 2: Blob store API
+
+You can use the BioSolr jar as a runtime library stored in the blob store API. In order to enable the use of runtime libraries, you must start your Solr instance with the flag `-Denable.runtime.lib=true`. **This option is now deprecated in solr 8 and will not be available anymore in Solr 9.**
```

Comment:
Is `-Denable.runtime.lib=true` enabled by default in Solr 8?

Reply:
Not that I'm aware of, but it is no longer needed when using the packages system (or it is set to true by default). It is not mentioned in the documentation for that usage and all works on the tests without setting it to true.